.gitignore

```
@@ -1,4 +1,6 @@
 node_modules/
-/build
-package-lock.json
+build/
+coverage/tmp/
```

```suggestion
/coverage/
```

It would be better if the entire coverage directory is not committed as part of the git history - the code coverage results are something that should be run and whose results viewed/ checked(e.g. as part of a CI pipeline), but not part of the repo itself.